src/main/java/de/konfidas/ttc/messages/logtime/LogTime.java

```diff
@@ -2,9 +2,16 @@
 
 public abstract class LogTime{
 
+
+
     public enum Type{
-        UNIX, GENERALIZED, UTC;
+        UNIX, GENERALIZED, UTC
+    }
+
+    public boolean wasNotAfter(LogTime logTime){
+        return this.getTime() <= logTime.getTime();

```

Comment:
Vorschlag: Ergänzen:
    public boolean isEqual(LogTime logTime){
        return this.getTime() == logTime.getTime();
    }

Das würde die Asserts bei den Tests etwas vereinfachen

Reply:
Good Idea :) thank you!

Reply:
done with commit `1c821434afcf57f9c93f6a7ff252fe2609c818ce`